docs/modules/ROOT/pages/how-tos/configuring/component-nudges.adoc

```diff
@@ -109,3 +109,23 @@ NOTE: You can also use **kubectl edit** to add the build-nudges-ref instead of p
 
 * Ensure that the existing references in the repositories of your components are correct.
 * Try running the necessary commands again, and ensure you have the correct syntax.
+
+== Using custom renovate config
+It is possible to configure your own custom renovate config, which will be used during nudging.
+Renovate config has to be specified in ConfigMap in your namespace.
+
+.There are two possible options:
+. create global config in ConfigMap named `global-nudging-renovate-config` which will be used for all nudged components in your namespace,
+unless component specific config exists
+. create config for specific nudged component in ConfigMap named `nudging-renovate-config-COMPONENT_NAME` and
+it will be used for the component in your namespace
+
+Both ConfigMaps have the same format, can contain `config.json` or `config.js` keys, if both specified `config.json` has precedence.
+In the value of the key is js or json in the string.
+
+When you have either of ConfigMaps, your config will be used unmodified and it is users responsibility to fix it when renovate pipeline will be failing because of it.
+
+.Examples
```

Comment:
Can we have more comprehensive example with auto-merging and/or commit message changed as requested by users.

Reply:
examples are only to show the formatting

for specific options, users should consult renovate docs